sqlstore/src/main/resources/db/migration/V8001__changelogs.sql

```diff
@@ -0,0 +1,188 @@
+ALTER TABLE BoxChangeLog ADD boxChangeLogId bigint(20) PRIMARY KEY AUTO_INCREMENT FIRST;
```

Comment:
I didn't know about `V8900__hibernatize_everything`. I don't think the change log updates will interfere with other changes though, so it should be okay in a separate file.